app/src/main/res/layout/faq_content.xml

```diff
@@ -14,7 +14,7 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@color/color_def_white">
+    android:background="@color/component_color_faq_activity_container_background_color">
```

Comment:
If this file is getting used in `FAQSingleActivity` then rename this colors too as per above comment.

Reply:
Done